twoliter/embedded/build.Dockerfile

```diff
@@ -401,21 +403,26 @@ ARG VARIANT
 ARG BYPASS_SOCKET
 ARG OUTPUT_SOCKET
 ARG BUILDER_UID
+ARG IN_PLACE_UPDATES
 ENV VARIANT=${VARIANT} VERSION_ID=${VERSION_ID} BUILD_ID=${BUILD_ID}
 WORKDIR /root
 
 USER root
 RUN --mount=target=/host \
     /host/build/tools/pipesys link --fd-socket "${BYPASS_SOCKET}" --target /bypass && \
     /host/build/tools/pipesys link --fd-socket "${OUTPUT_SOCKET}" --target /output && \
-    mkdir -p /local/migrations && \
+    mkdir -p /local/migrations; \
+    if [[ -n "${IN_PLACE_UPDATES}" ]]; then \
```

Comment:
nit: it'll make the diff annoying, but let's go ahead and indent the `if` and `else` block contents for readability. Especially the `else` clause suffers from trying to avoid it.